research-institute-for-nature-and-forest.csl

```diff
@@ -184,36 +205,32 @@
         </group>
       </if>
       <else-if type="bill book graphic legal_case legislation motion_picture report song thesis" match="any">
-        <group delimiter=", " prefix=". ">
+        <group delimiter=", ">
           <text macro="event"/>
           <text macro="publisher"/>
           <text variable="number-of-pages" prefix=" " suffix=" p"/>
```

Comment:
that's an old bug, but no need for the prefix here, given the space in the group delimiter above.

Reply:
Thanks for spotting this. Fixed in 6775cc4eb